tsconfig.json

```diff
@@ -64,7 +64,7 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
 
     /* Advanced Options */
-    // "skipLibCheck": true,                     /* Skip type checking of declaration files. */
+    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
```

Comment:
Don't like this setting here: had a "clash" with a global type (`ethereum`) defined in both: web3modal and walletlink, got the following error:

```
node_modules/walletlink/dist/index.d.ts:10:9 - error TS2687: All declarations of 'ethereum' must have identical modifiers.

10         ethereum?: WalletLinkProvider;
           ~~~~~~~~

node_modules/walletlink/dist/index.d.ts:10:9 - error TS2717: Subsequent property declarations must have the same type.  Property 'ethereum' must be of type 'any', but here has type 'WalletLinkProvider | undefined'.

10         ethereum?: WalletLinkProvider;
           ~~~~~~~~

  node_modules/web3modal/dist/components/Modal.d.ts:6:9
    6         ethereum: any;
              ~~~~~~~~
    'ethereum' was also declared here.

node_modules/web3modal/dist/components/Modal.d.ts:6:9 - error TS2687: All declarations of 'ethereum' must have identical modifiers.

6         ethereum: any;
          ~~~~~~~~


Found 3 errors.
```

There is an [open issue about this](https://github.com/microsoft/TypeScript/issues/37053)
I will test further but this might break installs of this library

Reply:
@tpae @kelokchan any ideas of how to resolve this issue without setting `skipLibCheck` to `true`?